lib/tasks/gettext.rake

```diff
@@ -90,4 +90,25 @@ namespace :gettext do
       end
     end
   end
+
```

Comment:
Can you add a description (`desc "does things"`) of this rake task so you see it in `rake -T`?